### DISCUSSION TITLE
How do you override icons ?

### DISCUSSION THREAD

**Discussion by zvrkSam** [LazyVim/LazyVim#4385] [Q&A] [answered]
I have seen that [#2707 ](https://github.com/LazyVim/LazyVim/discussions/2707) was once asked before and I have used that solution, but after some updates, this doesn't work anymore.
 I have tried all these options by themself, `overried`, `override_by_extension`, `override_by_filename` nothing works.
```lua
return {
  {
    "nvim-tree/nvim-web-devicons",
    opts = {
      strict = true,
      override = {
        go = {
          icon = "",
          color = "#81e043",
          name = "Go",
        },
      },
      override_by_extension = {
        ["go"] = {
          icon = "",
          color = "#81e043",
          name = "Go",
        },
      },
      override_by_filename = {
        [".go"] = {
          icon = "",
          color = "#81e043",
          name = "Go",
        },
      },
    },
  },
}
```
This is just a example me trying change one icon, I would like to change more then one.

**Comment by iamxiaojianzheng** [comment, marked as answer]
try this.

```lua
{
  "echasnovski/mini.icons",
  opts = {
    extension = {
      go = { glyph = "", hl = "MiniIconsYellow" },
    },
  },
},
```

Color reference 

https://github.com/echasnovski/mini.icons/blob/12e7b5d47bfc1b4c5ba4278fb49ec9100138df14/doc/mini-icons.txt#L150-L165

**Reply by zvrkSam** [reply]
Thank you, It works, I had a feeling something else was involved.

**Reply by iamxiaojianzheng** [reply]
The latest version of lazyvim replaces `nvim-web-devicons` with `mini-icons`, so the icons used by other related plugins are all derived from `mini.icons`.

That's what I remember.

**Reply by Alilacream** [reply]
Trully thank you so much for the answer, with it i was able to add php and go icons that i trully like (it's not going to change my work dev but at least kinda funny to have it ngl), i shall be waiting on the special icons for the special named sub directories (hooks, client, server etc..).
